lib/l10n/app_fr.arb

```
@@ -287,8 +287,8 @@
   "deleteAllDataWarning": "Cela supprimera définitivement tous vos chats, messages et paramètres de cet appareil. Cette action ne peut pas être annulée.",
   "deleteAllDataError": "Échec de la suppression de toutes les données. Veuillez réessayer.",
   "chatInformation": "Informations du chat",
-  "addAsContact": "Ajouter comme contact",
-  "removeAsContact": "Retirer des contacts",
+  "follow": "Suivre",
```

RuboCop (1.86.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)